### ISSUE TITLE
Refresh the status of compute node instance list when instance enabled or disabled

### ISSUE THREAD

**Issue by menghaoranss** [apache/shardingsphere#15966]
## Question

Now, all the compute nodes are persisted to `InstanceContext`, when we execute the following `DistSQL` to changed the status of instance: 

```
[enable / disable] instance [IP=xxx, PORT=xxx / instanceId]
```

the method ` InstanceContext.updateInstanceStatus` just refresh the status of current instance, the list of compute node should be refreshed to ensure that cluster state is consistent.

## How to fix
refer to `InstanceContext.updateLabel`

**Comment by soulasuna** [comment]
I can try assign to me please.

**Comment by menghaoranss** [comment]
fixes by #15991
